cfme/tests/test_replication.py

```diff
@@ -466,3 +470,210 @@ def test_replication_subscription_revalidation_pglogical(configured_appliance,
     region.replication.set_replication(replication_type="global",
                                        updates={"host": remote_app.hostname},
                                        validate=True)
+
+
+@pytest.mark.tier(2)
+@pytest.mark.ignore_stream("upstream")
+def test_replication_between_regions(provider, replicated_appliances):
+    """Test that a provider added to the remote appliance is replicated to the global
+    appliance.
+
+    Metadata:
+        test_flag: replication
+
+    Polarion:
+        assignee: dgaikwad
+        initialEstimate: 1/4h
+        casecomponent: Appliance
+    """
+    remote_appliance, global_appliance = replicated_appliances
+
+    remote_appliance.browser_steal = True
+    with remote_appliance:
+        provider.create()
+        remote_appliance.collections.infra_providers.wait_for_a_provider()
+
+    global_appliance.browser_steal = True
+    with global_appliance:
+        global_appliance.collections.infra_providers.wait_for_a_provider()
+        assert provider.exists
+
+
+@pytest.mark.tier(2)
+@pytest.mark.ignore_stream("upstream")
+def test_replication_database_disconnection(provider, replicated_appliances):
+    """Test that a provider created on the remote appliance *after* a database restart on the
+    global appliance is still successfully replicated to the global appliance.
+
+    Metadata:
+        test_flag: replication
+
+    Polarion:
+        assignee: dgaikwad
+        initialEstimate: 1/4h
+        casecomponent: Appliance
+    """
+    remote_appliance, global_appliance = replicated_appliances
+
+    global_appliance.db_service.stop()
+    sleep(60)
```

Comment:
Ehm... Can you put a comment explaining that we cannot use `wait_for`, perhaps on `global_appliance.db_service.is_active` or similar? if it is the case?

Reply:
@tpapaioa have you seen this comment? @jaryn is right here, please do explain the use of a 60s sleep over a wait_for condition.